src/components/Table/Table.stories.tsx

```diff
@@ -162,27 +155,25 @@ export const Stacked: StoryObj<Args> = {
     ),
   },
   parameters: {
-    viewport: {
-      defaultViewport: 'mobile2',
-    },
-    chromatic: { viewports: [414] },
+    // No hover functionality on chromatic, no need to snap.
+    chromatic: { disableSnapshot: true },
   },
 };
 
 export const AlignTableCellContentCenter: StoryObj<Args> = {
   args: {
-    caption: 'This is a table caption and it is required',
```

Comment:
Should we use the `TableCaption` in this story instead?

Reply:
Captions used to be [hidden by default](https://github.com/chanzuckerberg/edu-design-system/blob/ad1fb8a6b853a54b995cb3408cfaed079f6ab766/src/components/Table/Table.tsx#L62) so these captions in these stories were [never displayed](https://chanzuckerberg.github.io/edu-design-system/?path=/story/organisms-tables-table--align-table-cell-content-center).

However, I do think there is value in having a story with `TableCaption` to show its use, will add